KDE SVN revision r1230915.

Author: rkcosta (Raphael Kubo da Costa)
Date:   Sun May 8 17:30:06 2011 +0000

    Put the libarchive detection calls before configure_file().

    The generated config.h was using some variables defined only after
    plugins/CMakeLists.txt called macro_optional_find_package(LibArchive),
    however the top-level CMakeLists.txt relied on these variables already
    being set, which was never the case.

    BUG: 271553
    FIXED-IN: 4.6.4

--- ark/plugins/CMakeLists.txt
+++ ark/plugins/CMakeLists.txt
@@ -1,5 +1,3 @@
-macro_optional_find_package(LibArchive)
-
 if (LIBARCHIVE_FOUND)
     if( HAVE_LIBARCHIVE_READ_DISK_API )
         if( NOT HAVE_LIBARCHIVE_LZMA_SUPPORT OR NOT HAVE_LIBARCHIVE_XZ_SUPPORT )
@@ -15,8 +13,6 @@ if (LIBARCHIVE_FOUND)
     endif( HAVE_LIBARCHIVE_READ_DISK_API )
 endif (LIBARCHIVE_FOUND)
 
-macro_log_feature(LIBARCHIVE_FOUND "LibArchive" "A library for dealing with a wide variety of archive file formats" "http://code.google.com/p/libarchive/" FALSE "" "Required for among others tar, tar.gz, tar.bz2 formats in Ark.")
-
 add_subdirectory( clirarplugin )
 add_subdirectory( cli7zplugin )
 add_subdirectory( clizipplugin )
